ow_power_system/profiles/example_fault.csv

```diff
@@ -0,0 +1,51 @@
+Timestamp, power, temperature, voltage
```

Comment:
First column would be better labeled something like "Index" .  How the profile is read and explained to the user can be left for a followon Jira.

Reply:
Could as well label the second column "wattage", followed by "temperature (Celsius)", to make all the units clear.